outboundSender.go

```diff
@@ -601,11 +579,12 @@ func (obs *CaduceusOutboundSender) send(urls *ring.Ring, secret, acceptType stri
 	// find the event "short name"
 	event := msg.FindEventStringSubMatch()
 
+	//TODO: do this need to be replaced by the retry repo?
 	retryOptions := xhttp.RetryOptions{
 		Logger:   obs.logger,
 		Retries:  obs.deliveryRetries,
 		Interval: obs.deliveryInterval,
-		Counter:  obs.deliveryRetryCounter.With("url", obs.id, "event", event),
+		// Counter:  obs.deliveryRetryCounter.With("url", obs.id, "event", event), //webpa retry does not accept prometheus metrics
```

Comment:
we should replace `github.com/xmidt-org/webpa-common/v2/xhttp` with `https://github.com/xmidt-org/httpaux`

Reply:
cool thanks - there were a couple of webpa-common packages i couldn't find replacements for.